cmd/celestia-appd/cmd/init.go

```diff
@@ -0,0 +1,20 @@
+package cmd
+
+import (
+	"github.com/cosmos/cosmos-sdk/x/genutil/client/cli"
+	"github.com/spf13/cobra"
+
+	"github.com/celestiaorg/celestia-app/v3/app"
```

Comment:
test is failing b/c we just merged a PR to `main` that bumped to celestia-app v4 so this import path should likely be


```suggestion
	"github.com/celestiaorg/celestia-app/v4/app"
```